components/core/src/clp/ffi/ir_stream/utils.cpp

```diff
@@ -29,4 +30,23 @@ auto serialize_metadata(nlohmann::json& metadata, std::vector<int8_t>& ir_buf) -
 
     return true;
 }
+
+auto serialize_string_packet(std::string_view str, std::vector<int8_t>& buf) -> bool {
```

Comment:
Should be `serialize_string` to be consistent with the naming of other methods, right?